src/plugins/intel_npu/src/plugin/npuw/serialization.cpp

```diff
@@ -341,7 +345,7 @@ void ov::npuw::s11n::write_weightless(std::ostream& stream, const std::vector<ov
 
 void ov::npuw::s11n::read_weightless(std::istream& stream,
                                      std::vector<ov::Tensor>& var,
-                                     const ov::npuw::s11n::Weights& weights) {
+                                     const ov::npuw::s11n::CompiledDescDeserializeContext& ctx) {
```

Comment:
Not sure what's the difference here but I liked the previous apporach more. Why we now do the weightless read into the `CompiledDescDeserializeContext`? Sounds way too specific to me

Reply:
Renamed. Also we read from the ctx